### ISSUE TITLE
Issues with Module Decomposition Section of MG

### ISSUE THREAD

**Issue by samm82** [DeeshaPatel/CAS-741-Solar-Cooker#33]
- [x] The Constant Value Module doesn't use the values that it stores; either remove the second part of the description of its services or rephrase it to say that it is responsible for "providing them accordingly".
- [x] The Constant Value Module seems to be an Abstract Object instead of a Record; there is only one that exists for the life of the program, not multiple that get instantiated with different values.
- [x] What is the difference between the secret of the Input Format and Input Parameter Modules? The first seems like it just defines the structure for storing them (so its secret is correct) but the second seems like it actually stores these values (so its secret is incorrect).
- [x] Your Software Decision Modules should have more specific "Implemented By" fields; I don't think Python provides an ODE solver out of the box, so you should include which library you are using that actually provides it. (Your Sequence Data Structure Module is fine as is, since sequences _are_ provided by Python.)
- [x] The service of the Plotting Result Module should be rephrased. As is, it sounds like "solve it" is an input that is passed to the solver; I would say "Provides solvers that can take different inputs like **an** equation, **its** initial conditions, **a** time vector, **and** numerical parameters, and **use these to** solve ~it~ **the given equation**."

**Comment by DeeshaPatel** [comment]
@samm82 
One more thing associated with the MG, I have one question for you. 
Do you think the Control Module should be a library instead of Abstract Data Type?

**Comment by samm82** [comment]
Ooh good question. It doesn't make sense for it to be an ADT, since that implies that there could be more than one during the runtime of the program. Since the Control Module has state (at least according to the MIS), it should be an Abstract Object. If it didn't have state variables and just exported a set of functions for the user to use, then it would be a library.

**Comment by DeeshaPatel** [comment]
Thank you. :)
